_includes/links.md

```diff
@@ -35,7 +35,7 @@
 [pandoc]: https://pandoc.org/
 [paper-now]: https://github.com/PeerJ/paper-now
 [python-gapminder]: https://swcarpentry.github.io/python-novice-gapminder/
-[pyyaml]: https://pypi.python.org/pypi/PyYAML
+[pyyaml]: https://pypi.org//PyYAML
```

Comment:
Thanks for spotting this, Katrin. Could you please submit this change to carpentries/styles instead? Lessons are not supposed to be changing files in the `_includes` directory.

Reply:
I forget, sorry. See https://github.com/carpentries/styles/pull/457.